source/install/self-managed-changelog.md

```diff
@@ -70,7 +70,7 @@ Latest Mattermost Releases:
 ### Improvements
 
 #### User Interface (UI)
- - Persistent notifications allow users to notify recipients repeatedly until action is taken on an urgent message. Check out [our documentation](https://docs.mattermost.com/channels/message-priority.html#send-persistent-notifications) for more details.
+ - Persistent notifications (Professional and Enterprise Plans) allow users to notify recipients repeatedly until action is taken on an urgent message. Check out [our documentation](https://docs.mattermost.com/channels/message-priority.html#send-persistent-notifications) for more details.
```

Comment:
Thanks Amy, in the config settings section do we need to rearrange since it states "changes to team and enterprise edition"
![image](https://github.com/mattermost/docs/assets/14333569/bcb5b6d4-c4b7-4aed-a76c-adc0a0fad2a1)